### PR TITLE
another graphite examples that shows different things

### PR DESCRIPTION
**Pull request by Dieterbe**

### CODE REVIEW

**Comment by Dieterbe** [comment]
ping @kylebrandt

File: examples.md [hunk 1]
```diff
@@ -148,6 +148,60 @@ template header {
 }
 
 ~~~
+### Verifying and graphing per-host and aggregate available cpu capacity, using Graphite.
+This rule checks the current cpu capacity (avg per core, for the last 5 minutes), for each host in a cluster.
```
Comment (mjibson) [inline examples.md]: Need newlines between paragraphs.

**Comment by Dieterbe** [comment]
updated commit with extra newline before paragraph

File: examples.md [hunk 2]
```diff
@@ -149,6 +149,61 @@ template header {
 
 ~~~
 
+### Verifying and graphing per-host and aggregate available cpu capacity, using Graphite.
+This rule checks the current cpu capacity (avg per core, for the last 5 minutes), for each host in a cluster.
```
Comment (mjibson) [inline examples.md]: There should be newlines before and after each of these new paragraphs.

**Comment by Dieterbe** [comment]
amended commit with an extra new line after `### Verifying...` which I presume is what you meant

File: examples.md [hunk 3]
```diff
@@ -149,6 +149,62 @@ template header {
 
 ~~~
 
+### Verifying and graphing per-host and aggregate available cpu capacity, using Graphite.
+
+This rule checks the current cpu capacity (avg per core, for the last 5 minutes), for each host in a cluster.
```
Comment (mjibson) [inline examples.md]: And before and after here.

File: examples.md [hunk 4]
```diff
@@ -149,6 +149,62 @@ template header {
...
+We warn if more than 80% of systems have less than 20% capacity available, or if there's less than 200% in total in the cluster, irrespective of how many hosts there are and where the spare capacity is. Critical is for 90% and 50% respectively.
```
Comment (mjibson) [inline examples.md]: And before and after here.

File: examples.md [hunk 5]
```diff
@@ -149,6 +149,62 @@ template header {
...
+By checking a value that scales along with the size of the cluster, and one that puts an absolute lower limit, we can cover more ground.  But this is of course specific to the context of the environment and usage patterns.  Note that via groupByNode() the timeseries as returned by Graphite only have the hostname in them.
```
Comment (mjibson) [inline examples.md]: And before and after here.

File: examples.md [hunk 6]
```diff
@@ -149,6 +149,62 @@ template header {
...
+To provide a bit more context to the operator, we also plot the last 10 hours on a graph in the alert
```
Comment (mjibson) [inline examples.md]: And before and after here.

**Comment by Dieterbe** [comment]
like this then? or do we need 2 empty lines above "Verifying.." ?

**Pull request merged**